internal/service/resourcepolicy/resource_migration_test.go

```diff
@@ -7,12 +7,6 @@ import (
 )
 
 func TestMigResourcePolicy_basic(t *testing.T) {
-	mig.SkipIfVersionBelow(t, "1.22.0") // this feature was introduced in provider version 1.21.0, plural data source schema was changed in 1.22.0
-
-	var description *string
-	if mig.IsProviderVersionAtLeast("1.32.0") {
-		description = descriptionPtr
-	}
-
-	mig.CreateAndRunTestNonParallel(t, basicTestCase(t, description))
```

Comment:
older versions require the env variable to be defined, since we completely remove the usages this test would start running again after 1.33.0 release. Don't see significant concern as this test has been passing up to yesterday in which it failed due to env var no configured ([reference](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/14481955552/job/40620641363)).